.github/workflows/pull-request.yml

```diff
@@ -27,3 +27,24 @@ jobs:
             feat
             chore
             refactor 
+
+      - uses: marocchino/sticky-pull-request-comment@v2
+        if: always() && (steps.lint_pr_title.outputs.error_message != null)
+        with:
+          header: pr-title-lint-error
+          message: |
+            Hey there and thank you for opening this pull request! 👋🏼
+
+            We require pull request titles to follow the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/) and it looks like your proposed title needs to be adjusted.
+
+            Details:
+
+            ```
+            ${{ steps.lint_pr_title.outputs.error_message }}
+            ```
+
+      - if: ${{ steps.lint_pr_title.outputs.error_message == null }}
+        uses: marocchino/sticky-pull-request-comment@v2
+        with:   
+          header: pr-title-lint-error
+          delete: true
```

Comment:
If you are trying to create a comment, it needs write permissions, and if you are giving write permissions to a workflow, it can't run on `pull_request` because someone could exfiltrate the token. You need to use `pull_request_target` with any write permissions.